scripts/show-cf-memory-usage.rb

```diff
@@ -35,23 +39,24 @@
 orgs.each { |org|
   quota_id = org['entity']['quota_definition_guid']
   quota_def = quotas.select { |quota| quota['metadata']['guid'] == quota_id }[0]
-  orgs_reserved_memory += quota_def['entity']['memory_limit'].to_i
+
+  org_memory_quota = quota_def['entity']['memory_limit'].to_i
+  orgs_reserved_memory += org_memory_quota if org_memory_quota > 0
+
   allocated_services += quota_def['entity']['total_services'].to_i
   allocated_routes += quota_def['entity']['total_routes'].to_i
+
   org_id = org['metadata']['guid']
-  org_apps_reserved_memory = JSON.load(`cf curl /v2/organizations/#{org_id}/memory_usage`)['memory_usage_in_mb']
-  apps_reserved_memory += org_apps_reserved_memory.to_i
-  puts "Memory reserved by apps in org '#{org['entity']['name']}': #{org_apps_reserved_memory} MB"
+  org_apps_reserved_memory = JSON.load(`cf curl /v2/organizations/#{org_id}/memory_usage`)['memory_usage_in_mb'].to_i
+  apps_reserved_memory += org_apps_reserved_memory
+
+  puts "Org '#{org['entity']['name']}': quota: #{format_memory(org_memory_quota)}, reserved by apps: #{format_memory(org_apps_reserved_memory)}"
```

Comment:
It works. I'm getting:

```
Org 'admin': quota: 10240 MB (10 GB), reserved by apps: 4068 MB (3 GB)
```

I wonder if it reads more naturally to represent a fraction, such as:

```
Org 'admin': 4068 MB (3 GB) reserved by apps / 10240 MB (10 GB) quota
```